tutorials/g4g-public-health-ug/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Earth Engine for Public Health Land Surface Temperature Example Uganda
```

Comment:
No need to mention Earth Engine in the title, since it will already be under EE docs.

Reply:
Thanks for your comment - I have completed